### ISSUE TITLE
Can not run custom model

### ISSUE THREAD

**Issue by Artem-N** [thanhtantran/rknn-multi-threaded-3588#2]
Hi, have a problem - when start my own custom rknn model have a trouble, when i start one of models -   File "/home/orangepi/Desktop/rknn/rknn-multi-threaded-3588/func.py", line 208, in myFunc
    input1_data = outputs[1].reshape([3, -1]+list(outputs[1].shape[-2:]))
IndexError: list index out of range

and if start another model -   File "/home/orangepi/Desktop/rknn/rknn-multi-threaded-3588/func.py", line 207, in myFunc
    input0_data = outputs[0].reshape([10, -1]+list(outputs[0].shape[-2:]))
ValueError: cannot reshape array of size 2142000 into shape (10,newaxis,25200,85)

help please, think thats problem with model which i converted in onnx and then in rknn (first native yolo library for onnx, than rknn_model_zoo for rknn converting)

**Comment by Artem-N** [comment]
> Hi, have a problem - when start my own custom rknn model have a trouble, when i start one of models - File "/home/orangepi/Desktop/rknn/rknn-multi-threaded-3588/func.py", line 208, in myFunc input1_data = outputs[1].reshape([3, -1]+list(outputs[1].shape[-2:])) IndexError: list index out of range
> 
> and if start another model - File "/home/orangepi/Desktop/rknn/rknn-multi-threaded-3588/func.py", line 207, in myFunc input0_data = outputs[0].reshape([10, -1]+list(outputs[0].shape[-2:])) ValueError: cannot reshape array of size 2142000 into shape (10,newaxis,25200,85)
> 
> help please, think thats problem with model which i converted in onnx and then in rknn (first native yolo library for onnx, than rknn_model_zoo for rknn converting)

**Comment by programmeddeath1** [comment]
Hey how were you able to solve this? can you share some steps? Also what FPS did you achieve with your custom model?

**Comment by Artem-N** [comment]
So i had to struggle to get everything running as it should. Road the next - first we need convert our custom model from .pt to .onnx, then from .onnx to .rknn for our board on rockchip (in my case it orange pi 5 with rk3588), i used tolov7 for training.

My key mistake was that I converted to .onnx using the usual yolov7 repo, and it was necessary to convert using an airrockchip (https://github.com/airockchip) yolov7 repo to convert into  .onnx my custom .pt model.

Then in order to convert from .onnx to .rknn I installed on my PC with Windows ubuntu 20.04 and with the help of an airrockchip repo (https://github.com/airockchip/rknn_model_zoo in example/yolov7), I converted it to .rknn.

Then I loaded this model (.rknn) into my board, install req from https://github.com/airockchip/rknn-toolkit2 (warning - follow the version of what you install), install this repo. 
Then changed the path to the model in main.py, changed the classes to custom and anchors to anchors yolov7 (its bigger than yolov5) in func.py

i had 20/22 FPS with TPEs=3, not yet increased TPEs>3 due to temperature, haven`t heat sinks on chip.
